bundle/Resources/config/default_parameters.yml

```diff
@@ -1,4 +1,5 @@
 parameters:
+    netgen_remote_media.parameters.upload_prefix: 'https://api.cloudinary.com'
```

Comment:
If this parameter is defined through semantic config, then you don't need it defined here